tests/test_base.py

```diff
@@ -0,0 +1,54 @@
+from api import models
+from django.test import Client
+from django.contrib.auth import login
+from django.test import TestCase, Client
+from rest_framework.test import force_authenticate, APIRequestFactory, APIClient
+from django.urls import reverse
+from .factories import UserFactory, ItemFactory, GroupFactory, CollectionFactory, MeetingFactory
+
+
+class SimpleTest(TestCase):
```

Comment:
This will need a more descriptive name as we add in more tests. I believe the OSF API was splitting up the test classes based on views, e.g. `TestListCreateCollection`